scRNA-seq-advanced/03-dataset_integration.Rmd

```diff
@@ -276,37 +293,41 @@ purrr::map(sce_list,
 #### Ensuring that only shared genes are used
 
 The next step in ensuring SCE compatibility is to make sure they all contain the same genes, which are stored as the SCE object's row names (these names are also found the `rowData` slot's row names). 
-Here, those gene ids are unique Ensembl gene ids.
+Here, those gene IDs are unique Ensembl gene IDs.
 
 We can use some `purrr` magic to quickly find the set of shared genes among our samples:
 
-```{r shared genes}
-shared_genes <- sce_list %>%
+```{r shared_genes, live = TRUE}
+# Define vector of shared genes
+shared_genes <- sce_list |>
   # get rownames (genes) for each SCE in sce_list
-  purrr::map(rownames) %>%
+  purrr::map(rownames) |>
   # reduce to the _intersection_ among lists
   purrr::reduce(intersect)
+```
 
-# We've now created a vector of genes that are present in all SCEs
+```{r print_shared_genes, live = TRUE}
+# Use head to look at the vector of shared genes:
 head(shared_genes)
 ```
 
 In this case, we happen to know that all SCE objects we're working with already contained the same genes. 
-We can quickly confirm that this is true by looking at the number of rows across SCE objects, and we'll see that they are all the same:
+We do a quick-and-dirty check for this by looking at the number of rows across SCE objects, and we'll see that they are all the same:
 
-```{r check shared genes}
+```{r check_shared_genes, live = TRUE}
 # The number of genes in an SCE corresponds to its number of rows:
-sce_list %>%
+sce_list |>
   purrr::map(nrow)
 ```
 
 So, for our data, we will not have to subset to shared genes since they are already shared!
  
 #### Ensuring matching columns in `colData`
 
-Finally, we'll need to have the same column names across all SCE `colData` tables, so let's look at all those column names:
+Finally, we'll need to have the same column names across all SCE `colData` tables, so let's look at all those column names.
+We can use similar syntax here to what we used to look at all the `rowData` column names.
 
-```{r compare coldata}
+```{r compare_coldata, live = TRUE}
 purrr::map(sce_list,
            ~colnames(colData(.)))
```

Comment:
If we want to be consistent about `purrr` syntax between notebooks: (we would need to change the rowData chunk too).

```suggestion
           \(x) colnames(colData(x)) )
```